aip_x2_gen2_launch/config/polar_voxel_outlier_filter_node.param.yaml

```diff
@@ -0,0 +1,22 @@
+/**:
+  ros__parameters:
+    publish_noise_cloud: false                             # Publish a separate point cloud containing detected noise points
+    radial_resolution_m: 0.5                              # Voxel grid resolution in the radial (distance) direction, in meters
+    azimuth_resolution_rad: 0.0175                        # Voxel grid resolution in the azimuth (horizontal angle) direction, in radians (~1 degree)
+    elevation_resolution_rad: 0.0175                      # Voxel grid resolution in the elevation (vertical angle) direction, in radians (~1 degree)
+    voxel_points_threshold: 1                             # Minimum number of points required in a voxel to be considered valid
+    min_radius_m: 0.5                                     # Minimum distance from the sensor to include points, in meters
+    max_radius_m: 300.0                                   # Maximum distance from the sensor to include points, in meters
+    visibility_estimation_max_range_m: 10.0               # Maximum range for estimating point cloud visibility, in meters
+    visibility_estimation_max_secondary_voxel_count: 300  # Max number of secondary voxels to consider in visibility estimation
+    visibility_estimation_only: true                      # If true, only perform visibility estimation without filtering
+    use_return_type_classification: true                  # Use return type information to classify points (e.g., primary/secondary returns)
+    filter_secondary_returns: false                       # If true, filter out secondary returns and keep only primary returns
+    secondary_noise_threshold: 2                          # Threshold for classifying noise in secondary returns
+    intensity_threshold: 1                                # Minimum intensity value for a point to be considered valid
+    primary_return_types: [1, 6, 8, 10]                   # List of return types considered as primary returns
+    visibility_warn_threshold: 0.7                        # Warning threshold for visibility diagnostics
+    filter_ratio_error_threshold: 0.5                     # Error threshold for filter ratio diagnostics
+    filter_ratio_warn_threshold: 0.7                      # Warning threshold for filter ratio diagnostics
+    visibility_error_threshold: 0.8                       # Error threshold for visibility diagnostics
+    visibility_warn_threshold: 0.9                        # Warning threshold for visibility diagnostics
```

Comment:
@manato 
Same parameter with L18?

Reply:
@YoshiRi 
Thank you for pointing this out!

@drwnz @linick-kang 
Regarding `visibility_warn_threshold`, duplicate configuration was found in [the evaluation branch](https://github.com/tier4/aip_launcher/commit/179fe5b3f9128b6e155096b69be62b00c476cd09).  Could you share which value is intended to set?;  `0.7` or `0.9`

Reply:
fixed at [aa66b4d](https://github.com/tier4/aip_launcher/pull/557/commits/aa66b4da7b457806c22e81542fa3417231085466)